### ISSUE TITLE
The second run of `eslint-baseline` doesn't show errors

### ISSUE THREAD

**Issue by maks-rafalko-unagi** [lukahartwig/eslint-baseline#13]
According to the readme, I've executed

```bash
npx eslint-baseline src
```

the command above (1st run) generated baseline file.

Then I on purpose change the code so it has an error. Run again

```bash
npx eslint-baseline src
```

and nothing happens. It returns `0` exit code and doesn't display the new error.

When I try to update baseline by

```bash
npx eslint-baseline src --update-baseline
```

the new error is being added. So I suppose something is broken.

Tried to get back to 0.3.2, but it fails with https://github.com/lukahartwig/eslint-baseline/issues/11
